tests/aws/services/kms/test_kms.py

```diff
@@ -793,6 +793,88 @@ def test_import_key_asymmetric(self, kms_create_key, aws_client, snapshot):
         describe_key_after_deleted_import = aws_client.kms.describe_key(KeyId=key_id)
         snapshot.match("describe-key-after-deleted-import", describe_key_after_deleted_import)
 
+    @markers.aws.unknown
```

Comment:
I've cross-verified the logic in this PR with an external client and a real AWS KMS instance, so I'm confident this behaves like the original AWS API.
However, I have some troubles running the parity-validation steps there at this time. After merging this PR, it would be useful if some maintainer could run it there and switch this to a `@markers.aws.validated`. Thanks!